nncf/common/accuracy_aware_training/training_loop.py

```diff
@@ -173,7 +169,30 @@ def _accuracy_criterion_satisfied(accuracy_budget, compression_controller):
                compression_controller.compression_stage() == CompressionStage.FULLY_COMPRESSED
 
 
-class AdaptiveCompressionTrainingLoop(TrainingLoop):
+class EarlyExitCompressionTrainingLoop(BaseEarlyExitCompressionTrainingLoop):
+    """
+    Adaptive compression training loop allows an accuracy-aware training process
+    to reach the maximal accuracy drop
+    (the maximal allowed accuracy degradation criterion is satisfied).
+    """
+
+    def __init__(self,
+                 nncf_config: NNCFConfig,
+                 compression_controller: CompressionAlgorithmController,
+                 lr_updates_needed=True, verbose=True,
+                 dump_checkpoints=True):
```

Comment:
Set `bool` type hints here as well.

Reply:
It was decided to improve code quality within a separate task